src/rpc/server.cpp

```diff
@@ -19,7 +19,7 @@ void Server::register_service(grpc::Service* service) {
 
 void Server::start() {
     if (server_) {
-        throw "Attempted to start server that was already running";
+        throw std::runtime_error("Attempted to start server that was already running");
```

Comment:
(driveby) adding documentation, wanted to throw an actual error rather than a string. easy enough to fix!